docs/start/building.md

```diff
@@ -50,7 +50,11 @@ mvn clean install -DskipTests
 
 This instructs [Maven](http://maven.apache.org) (`mvn`) to first remove all existing builds (`clean`) and then create a new Flink binary (`install`).
 
-To speed up the build you can skip tests, checkstyle, and JavaDocs: `mvn clean install -DskipTests -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true`.
+To speed up the build you can skip tests, checkstyle, and JavaDocs:
```

Comment:
replace `checkstyle` with `QA-plugins`

Reply:
What's `QA-plugins`? Do you mean the set of QA checks in `tools/qa-check.sh`?

Reply:
quality-assurance plugins, like checkstyle, rat, japicmp, enforcer etc. . Guess we'll need an easier to understand name :/